app/Entirety/entities/views.py

```diff
@@ -248,23 +264,13 @@ def post(self, request, *args, **kwargs):
                         i = i + 1
                     j = j + 1
                 res = post_entity(self, entity, False, self.project)
-                if res:
```

Comment:
@sbanoeon, this is duplicated, right?

Reply:
This part of code is delete.

Reply:
Yes, I deleted it yesterday. I would like to know your opinion

Reply:
Oh okay, let me test and have a look at that.

Reply:
Yes, indeed, it was duplicated.